NewForce-Cohort-9/nutshell-react-steer-clear
# Ks branch

- have ugly messages displaying
- have create message working

to test:
- go to ks-branch
- see that localhost:5173/messages displays messages (there is no button on this page yet to create a new message)
- go to localhost:5173/newMessage and try posting a message
